packages/ERTP/src/issuer.js

```diff
@@ -170,12 +176,12 @@ function makeIssuerKit(
       });
     }
 
-    const total = payments.map(paymentLedger.get).reduce(add, empty);
+    const total = payments.map(paymentLedger.get).reduce(add, emptyAmount);
 
-    const newTotal = newPaymentBalances.reduce(add, empty);
+    const newTotal = newPaymentBalances.reduce(add, emptyAmount);
```

Comment:
I like how you use `emptyAmount` to ensure that there's always agreement on the mathKind.